tests/kola/test-bootupd

```diff
@@ -115,6 +117,9 @@ bootupctl update | tee out.txt
 assert_file_has_content_literal out.txt 'No update available for any component'
 assert_not_file_has_content_literal out.txt 'Updated EFI'
 
+# Ensure /boot and /boot/efi can be written to
+mount -o remount,rw ${bootmount}
+mount -o remount,rw ${bootefimount}
```

Comment:
This is OK for now, but in the future we can clean this up by having the test do something like re-exec itself in a new mount namespace so we're not affecting the main one.